packages/frontend/src/Build.spec.ts

```diff
@@ -60,6 +60,7 @@ const mockBootcImages: ImageInfo[] = [
     Size: 0,
     Containers: 0,
     SharedSize: 0,
+    Digest: 'sha256:image1',
```

Comment:
Pinged Charlie to confirm - updating PD API causes Digest to be a required field, hence the need to add it here and other places.